examples/policies/multi-asset-policy/multi-asset-policy.pact

```diff
@@ -0,0 +1,337 @@
+(namespace (read-msg 'ns))
+
+(module multi-asset-policy GOVERNANCE
+
+  @doc "Policy for context-dependant multi-asset tokens."
+
+  (implements kip.token-policy-v2)
+
+  (use kip.token-manifest)
+  (use kip.token-policy-v2 [token-info])
+  (use marmalade-v2.policy-manager)
+  (use marmalade-v2.non-fungible-policy-v1)
+
+  (defconst ADMIN-KS:string "marmalade-examples.multi-asset-policy")
+
+  (defcap GOVERNANCE ()
+    (enforce-guard ADMIN-KS))
+
+  (defconst OPERATOR-GUARD-MSG-KEY:string "operator_guard")
+  (defconst ASSETS-MSG-KEY:string "assets")
+
+  (defcap ASSET_PROPOSED (token-id:string asset-id:integer uri:string operator-guard:guard)
+    @doc "Emitted when new asset is proposed"
+    @event
+
+    (enforce-guard operator-guard)
+  )
+  
+  (defcap ASSET_REJECTED (token-id:string asset-id:integer uri:string owner:string)
```

Comment:
I think it may be better to separate events and signing capabilities, if we want to let users to be able to call `reject-all-proposed-assets` without looking up `asset-id`, and `uri`. Even for `reject-proposed-asset` function, the user only needs to provide `token-id, asset-id, owner` parameters, but they have to provide `uri` string in the capability. 

I would recommend `(REJECT_ALL_ASSET token-id owner)` function for signing `reject-all..`, and `(REJECT_ASSET token-id asset-id owner)` for `reject-proposed-asset` for signature, and then emit the `ASSET_REJECTED` at the end.